Dockerfile

```diff
@@ -1,36 +1,7 @@
-FROM ubuntu:14.04
-MAINTAINER Colin Alston <colin@praekelt.com>
-RUN apt-get update && apt-get -y --force-yes install libjpeg-dev zlib1g-dev libxslt1-dev libpq-dev nginx supervisor python-dev python-pip libffi-dev
-RUN apt-get -y install postgis
+FROM praekeltfoundation/django-bootstrap
 
-RUN pip install --upgrade pip
+RUN apt-get-install.sh gettext
 
-ENV PROJECT_ROOT /deploy/
-ENV DJANGO_SETTINGS_MODULE service_directory.project.docker
+ENV DJANGO_SETTINGS_MODULE=service_directory.settings.docker
```

Comment:
Looks like it'd be easiest to copy `docker/settings.py` to `service_directory/project/docker.py` or something and leave this setting as is.